src/components/PackTile.js

```diff
@@ -73,95 +73,134 @@ const PackTile = ({
       className={className}
       interactive
       css={css`
+        --tile-image-height: 100px; /* Logo image height */
```

Comment:
Was having issues with this, seeing the icon pop outside of the white background, was able to fix it by changing this to `100%`

<img width="263" alt="Screen Shot 2022-01-06 at 2 45 27 PM" src="https://user-images.githubusercontent.com/26727669/148449631-00e56364-acc3-4aaf-ab6d-f54496af3605.png">

Reply:
This is on grid view, desktop

Reply:
This should be fixed with the latest commit!